demos/redpanda/ockam-setup

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+NAME="kafkabroker"
+
+ockam reset
+ockam enroll
+
+OCKAM_BROKER_TOKEN=$(ockam project ticket --attribute component=kafkabroker)
```

Comment:
we should all these tickets, this value is no longer just a token

Reply:
`--attribute component=kafkabroker` is unless we plan to set policies based on that attribute